components/resources/brave_components_strings.grd

```diff
@@ -433,11 +433,17 @@
       <if expr="not is_android">
         <message name="IDS_BRAVE_WEBCOMPATREPORTER_REPORT_MODAL_TITLE" desc="Title for broken website report dialog window">Report a broken site</message>
         <message name="IDS_BRAVE_WEBCOMPATREPORTER_REPORT_EXPLANATION" desc="Label for a website's URL indicating that it will be reported for compatibility issues">Let Brave's developers know that this site doesn't work properly with Shields:</message>
-        <message name="IDS_BRAVE_WEBCOMPATREPORTER_REPORT_DISCLAIMER" desc="Describes what information is sent to Brave's servers in a broken website report">Note: The report sent to Brave servers will include the site address, Brave version number, Shields settings, VPN status, and language settings.</message>
+        <message name="IDS_BRAVE_WEBCOMPATREPORTER_REPORT_INTERNAL_EXPLANATION" desc="Notifies the user that internal browser pages cannot be reported">This page is an internal browser page and is not eligible for reporting.</message>
```

Comment:
this message isn't necessarily true because file://, data://, etc. will trigger this error (since they aren't http/https) and are not internal browser pages. suggest changing it to just `This page is not eligible for reporting`

Reply:
please see `IDS_BRAVE_WEBCOMPATREPORTER_REPORT_LOCAL_EXPLANATION`

Reply:
@DJAndries not sure i understand. i get that LOCAL_EXPLANATION is shown for `isLocalPage`, which is fine. i'm saying that INTERNAL_EXPLANATION doesn't fully make sense for `!isHttpPage` because there are many non-HTTP(s) schemes which are NOT a "internal browser page". to me "internal browser page" would only be brave://.

Reply:
sorry, i didn't read this properly. changed the copy to:

```
This page cannot be reported because it was not served over HTTP/HTTPS.
```

Reply:
sgtm